relay-ourlogs/src/ourlog.rs

```diff
@@ -131,76 +131,59 @@ pub fn otel_to_sentry_log(otel_log: OtelLog) -> OurLog {
 pub fn ourlog_merge_otel(ourlog: Annotated<OurLog>) -> Annotated<OurLog> {
     let mut ourlog = ourlog;
     if let Some(ourlog_value) = ourlog.value_mut() {
-        if let Some(attributes) = ourlog_value.attributes.value_mut() {
-            attributes.insert(
-                "sentry.severity_number".to_string(),
-                Annotated::new(AttributeValue::IntValue(level_to_otel_severity_number(
-                    ourlog_value.level.value().cloned(),
-                ))),
-            );
-            attributes.insert(
-                "sentry.severity_text".to_string(),
-                Annotated::new(AttributeValue::StringValue(
-                    ourlog_value
-                        .level
-                        .value()
-                        .map(|level| level.to_string())
-                        .unwrap_or_else(|| "info".to_string()),
-                )),
-            );
-        }
+        let attributes = ourlog_value.attributes.value_mut().get_or_insert_default();
+        attributes.insert(
+            "sentry.severity_number".to_string(),
+            Annotated::new(AttributeValue::IntValue(level_to_otel_severity_number(
+                ourlog_value.level.value().cloned(),
+            ))),
+        );
+        attributes.insert(
+            "sentry.severity_text".to_string(),
```

Comment:
```suggestion
            "sentry.severity_text".to_owned(),
```

We generally use `to_owned` for `&str` -> `String`. It expresses the intent better and it doesn't go through the `Display` trait to do the conversion.